templates/org_mozilla_fenix/clients_last_seen/view.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.org_mozilla_fenix.clients_last_seen` AS
+SELECT
+  -- Include date_last_seen for compatibility with existing queries.
+  DATE_SUB(submission_date, INTERVAL days_since_seen DAY) AS date_last_seen,
+  -- We cannot use UDFs in a view, so we paste the body of udf_bitpos(bits) literally here.
+  CAST(SAFE.LOG(days_seen_bits & -days_seen_bits, 2) AS INT64) AS days_since_seen,
+  CAST(SAFE.LOG(days_created_profile_bits & -days_created_profile_bits, 2) AS INT64) AS days_since_created_profile,
+  *
+FROM
+  `moz-fx-data-derived-datasets.org_mozilla_fenix.clients_last_seen_v1`
```

Comment:
I don't see `org_mozilla_fenix.clients_last_seen_v1` defined in this PR, but it seems like these columns should be defined in a view named `org_mozilla_fenix.clients_last_seen_v1` that reads from `org_mozilla_fenix_derived.clients_last_seen_v1`, while `org_mozilla_fenix.clients_last_seen` would be created by `generate_and_publish_views`

Reply:
My standpoint is that going forward, we generally do not want include a version suffix on user-facing views unless there is a specific need to. Similar to how we only present `telemetry.core` as a union of the underlying tables, I expect users to interact with this data as `org_mozilla_fenix.clients_last_seen`.

I had imagined that the "generate" portion of `generate_and_publish_views` would only apply to `_stable` tables. For derived tables, there is not necessarily a 1:1 relationship between underlying tables and user-facing views, so my assumption is that we always write an explicit view and check it into this repo.

Reply:
lgtm

Reply:
docs issue filed for later at #343